hack/generated/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,28 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for
+# breaking changes
+
+# TODO: The above comments are a lie
```

Comment:
![gif](https://media.giphy.com/media/ZqlvCTNHpqrio/giphy.gif)

Reply:
Hah I should fix this before merge. I think we use certmanager like 1.4 now so just need to fix that bit.